packages/cli/README.md

```diff
@@ -16,260 +16,9 @@ Or add as a package dependency:
 npm install @medplum/cli
 ```
 
-## Authentication
```

Comment:
@jamestouri - I hope this is ok.  Most of this doc was duplicated in `docs/cli/index.md`, so I just added a link to that page.